config/crds/migration.openshift.io_migplans.yaml

```diff
@@ -525,16 +529,25 @@ spec:
                     is the default AccessModes - access modes supported by the dynamic
                     provisioner
                   properties:
-                    accessModes:
-                      items:
-                        type: string
-                      type: array
                     default:
                       type: boolean
                     name:
                       type: string
                     provisioner:
                       type: string
+                    volumeAccessModes:
```

Comment:
this could be a upgrade problem...how does this affect migration plans that are already present in a cluster from an older version?

Reply:
So the field is optional, so an existing migration plan should not have a problem. The values will just be empty.

Reply:
@awels but I see that accessModes field is removed and being replaced with this

Reply:
Good point, I should probably preserve that, and make an internal function to simply convert that to fs since that is what it was originally.

Reply:
So looking at this some more, it doesn't appear to be used as a K8S resource at all. So no existing plans should be affected by it. The struct is only used when looking up information for storage classes, and then it is dynamically populated.

Reply:
@awels okay so I taking your word for it on this. I don't know/remember enough to say 100% that this will work. But if you can test an upgrade scenario from a migration plan that has this field to a newer version...and it works, I am good with the change.

Reply:
Well there are no migration plans with this field since it is not stored at all. That is why it is not a problem.